lists-service/cicd/base/config.ini

```diff
@@ -15,6 +15,9 @@ SLACK_DEPLOY_NOTIFICATION = true
 SLACK_ALERT_CHANNEL = zabbix-lists
 AUTO_DEPLOY = true
 
+COGNITO_STACK_NAME = ala-cognito-pool-external-app-clients
+APP_CLIENT_DOMAIN = {PRODUCT_NAME}-${CLEAN_BRANCH}.${HOSTED_ZONE}
```

Comment:
is this a duplicate of the app client domain used in the deploy component?

Reply:
it's not duplicate same domain from the deploy component, however is is the same domain as the lists-ui frontend component. These domains are similar (ie. lists.ala.org.au vs lists-ws.ala.org.au).

On a side note: I believe the backend should be exposed by the api gateway, eg. https://api.ala.org.au/lists